client/src/ui/atoms/search/index.scss

```diff
@@ -31,7 +31,18 @@
     outline: var(--border-primary);
   }
 
+  &:not([value=""]) {
+    ~ .button.clear-search-button {
+      display: block;
+      right: 2rem;
```

Comment:
This reduced margin (from `2.8rem`) breaks the search box at the top of the page:

![Screen Shot 2023-02-03 at 16 49 45](https://user-images.githubusercontent.com/755354/216660436-d7929d4a-71ff-4f43-ba73-bd2e75653fd2.png)

Reply:
Fixed in 6dcbd91eb.